### PR TITLE
update http module with requests, methods, and statuses

### PR DESCRIPTION
**Pull request by briandowns**
Signed-off-by: Brian Downs <brian.downs@gmail.com>

### Well detailed description of the change :

Resolves: 
 
#

### Type of change:

Adds access to common headers, response codes, and messages.

- [ ] Bug fix

- [x] New feature

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#

### Housekeeping

- [ ] Tests have been updated to reflect the changes done within this PR (if applicable).

- [ ] Documentation has been updated to reflect the changes done within this PR (if applicable).

### CODE REVIEW

**Comment by briandowns** [comment]
I can update docs if this changes is deemed acceptable.

File: tests/http/status.du [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import HTTP;
+
+assert(HTTP.STATUS_OK == 200);
```
Comment (Jason2605) [inline tests/http/status.du:3]: Remember to add this file to: https://github.com/dictu-lang/Dictu/blob/develop/tests/http/import.du

Other than that looks fine to me! I wonder how we can add all of these to the docs without making the page too unmanageable.

Comment (briandowns) [reply tests/http/status.du:3]: I took a stab at some verbiage to describe the added functionality in the docs. If you want it very explicit, I'm happy to do so.

Comment (Jason2605) [reply tests/http/status.du:3]: I'm pretty happy with that to be honest. I think listing out every constant on the page would be a bit much, thanks!

**Pull request merged**